src/components/pages/Dashboard/ServiceMap.js

```diff
@@ -15,13 +15,50 @@ export default function ServiceMap(props) {
     zoom: 11,
     bearing: 0,
     pitch: 0,
+    width: 5000,
+    height: 500,
   });
 
+  const [popupInfo, setPopupInfo] = useState(null);
+
   const mapRef = useRef(null);
 
-  const onClick = event => {
-    const feature = event.features[0];
-    console.log('features', feature);
+  const updateViewport = viewport => {
+    setViewport(viewport);
+  };
+
+  const renderMarker = (location, index) => {
+    return (
+      <Marker
+        key={`marker-${index}`}
+        longitude={location.longitude}
+        latitude={location.latitude}
+        n
```

Comment:
theres an n here when it shouldn't but its not code breaking so next PR we should remember to remove it